client/ayon_core/addon/interfaces.py

```diff
@@ -38,10 +53,25 @@ class IPluginPaths(AYONInterface):
     """
 
     @abstractmethod
-    def get_plugin_paths(self):
-        pass
+    def get_plugin_paths(self) -> dict[str, list[str]]:
```

Comment:
We can't us `dict` and `list`.
```suggestion
    def get_plugin_paths(self) -> Dict[str, List[str]]:
```

Reply:
we can, because of `__future__.annotation`, it works even on python 3.7